packages/beacon-node/src/metrics/validatorMonitor.ts

```diff
@@ -306,21 +306,21 @@ export function createValidatorMonitor(
         case "active_exiting":
         case "active_slashed":
         case "active_ongoing":
-          metrics.validatorMonitor.validatorsActive.inc();
+          metrics.validatorMonitor.validatorStatus.inc({status: "active"});
```

Comment:
What are you tracking here exactly? The total number of counts validators with a specific status were seem in an epoch? Not sure this metrics represents the instantaneous count of connected validators with a specific status

Reply:
I am not sure I understand. This function would be called while iterating through an array of the validator indices, and each registered validator should have a status, therefore the metrics object would then increment each status.

Reply:
we want to track the status of local validators here right? and assuming `registerValidatorStatuses` is called every epoch we would just increment this indefinitely